packages/miniflare/src/index.ts

```diff
@@ -1239,6 +1252,7 @@ export class Miniflare {
 				unsafeEphemeralDurableObjects,
 				queueProducers,
 				queueConsumers,
+				unsafeEnableAssetsRpc,
```

Comment:
we need this so that the assets plugin can conditionally add the rpc proxy Worker